x-pack/plugins/observability_solution/apm/public/components/routing/templates/apm_main_template/index.tsx

```diff
@@ -101,6 +100,8 @@ export function ApmMainTemplate({
     location.pathname.includes(path)
   );
 
+  const isEntityExperienceFullyEnabled = isEntityCentricExperienceEnabled && isEntityManagerEnabled;
```

Comment:
Why do we need to care if `isEntityManagerEnabled` is enabled?

Only looking at the FF and the `serviceEntitySummary?.dataStreamTypes` is enough to bypass the no data screen. We know `serviceEntitySummary?.dataStreamTypes` will only be populated if EEM is enabled.